### PR TITLE
feat(security): secure-by-default baseline control module

### LINKED ISSUE #187: feat(security): secure-by-default baseline for first-run and day-2 operations
## Problem
Safe operation currently depends too much on manual user hardening decisions.

## Goal
Ship secure defaults so first-time users are protected without advanced tuning.

## Scope
- Least-privilege defaults for runtime/config operations.
- Sensitive value masking in logs/UI/diagnostics.
- Confirmation gates for destructive or high-risk actions.
- Fail-closed behavior on policy uncertainty.

## Out of Scope
- Enterprise IAM federation
- External KMS/Vault integration (unless already present)

## Functional Requirements
- Security settings visible and explainable in one place.
- Regression tests prevent accidental permission broadening.

## Acceptance Criteria
- Fresh install operates under secure defaults by default.
- Sensitive data is consistently redacted.
- High-risk actions require explicit user confirmation.

### PR DESCRIPTION
**Pull request by Akane-CN**
## Summary
Implements secure-by-default baseline logic for first-run and day-2 safety behavior.

## Changes
- Added explicit secure defaults (least-privilege mode, output masking, confirmation gates, fail-closed policy behavior)
- Added sensitive-value masking helper
- Added explicit confirmation requirement for high-risk actions
- Added fail-closed decision path when policy certainty is missing
- Added unit tests that guard against accidental default broadening

## Issue
Closes #187

## Acceptance Mapping
- **Fresh install secure by default**: baseline defaults are all secure-first and enabled.
- **Sensitive data consistently redacted**: masking helper is included and tested.
- **High-risk actions require explicit confirmation**: unconfirmed high-risk actions are blocked.

### CODE REVIEW

File: fiber-link-service/apps/admin/src/features/security/secure-defaults.test.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { describe, expect, it } from "vitest";
+import { SECURE_DEFAULTS, canRunHighRiskAction, evaluatePolicyDecision, maskSensitive } from "./secure-defaults";
+
+describe("secure defaults", () => {
+  it("ships secure baseline enabled by default", () => {
+    expect(SECURE_DEFAULTS.leastPrivilegeMode).toBe(true);
+    expect(SECURE_DEFAULTS.maskSensitiveOutput).toBe(true);
+    expect(SECURE_DEFAULTS.requireHighRiskConfirmation).toBe(true);
```
[the remark this answers is not in the document]
Comment (Akane-CN) [reply fiber-link-service/apps/admin/src/features/security/secure-defaults.test.ts]: Good catch — fixed in 66f5262. Added the missing assertion for `SECURE_DEFAULTS.failClosedOnPolicyUncertainty` in the baseline defaults test to guard against regressions.

File: fiber-link-service/apps/admin/src/features/security/secure-defaults.test.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  it("requires explicit confirmation for high-risk actions", () => {
+    expect(canRunHighRiskAction(false)).toBe(false);
+    expect(canRunHighRiskAction(true)).toBe(true);
+  });
+
+  it("fails closed when policy is uncertain and masks secrets", () => {
+    expect(evaluatePolicyDecision({ policyKnown: false })).toBe("deny");
+    expect(maskSensitive("abc123secretxyz")).toContain("***");
+  });
```
[the remark this answers is not in the document]
Comment (Akane-CN) [reply fiber-link-service/apps/admin/src/features/security/secure-defaults.test.ts]: Addressed in 42bff2a as part of test hardening: split policy and masking concerns, replaced weak `toContain` assertion with explicit expected outputs, and added short-string masking edge cases.
